pkg/packer/manifests/vsphere/packer.pkr.hcl

```diff
@@ -516,7 +511,6 @@ build {
       gpu_nvidia_version       = "${var.gpu_nvidia_version}"
       gpu_types                = "${var.gpu_types}"
       guest_os_type            = "${var.guest_os_type}"
-      ib_version               = "${var.ib_version}"
```

Comment:
Just curious, what was this?

Reply:
here's the link to the conversation in the original PR @msdolbey filed when he noticed it: https://github.com/mesosphere/konvoy-image-builder/pull/818#discussion_r1211998143

We weren't using it at all 🤷‍♀️

Reply:
yeah. we carried it over from upstream `image-builder` and didnt modified it. :)